mne/surface.py

```diff
@@ -1893,25 +1889,20 @@ def warp_montage_volume(montage, base_image, reg_affine, sdr_morph,
     _warn_missing_chs(montage, image_to, after_warp=True)
 
     # recover the contact positions as the center of mass
-    warped_data = _get_img_fdata(image_to)
+    warped_data = np.array(image_to.dataobj)
```

Comment:
No copy needed and native dtype is fine so `np.asanyarray` is probably better here